### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by MarinaT-N**

### CODE REVIEW

File: cypress/e2e/login.cy.js [hunk 1]
```diff
@@ -1,11 +1,71 @@
...
+
+    cy.get('#flash')
+    .should('contain.text','You logged into a secure area!');
+
+    cy.assertUrl('secure'); 
+
   });
 
-  it('', () => {
+  it('should does not allow USER to Login with invalid creds', () => {
```
Comment (NataliiaSolotva) [inline cypress/e2e/login.cy.js:26]:
```suggestion
  it('should not allow USER to Login with invalid creds', () => {
```

File: cypress/e2e/login.cy.js [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
+
+    const randomNuber = Math.random().toString().slice(2,10);
+    const username = `test${randomNuber}`;
     
+    const password = `${randomNuber}asd!Q`;
```
Comment (NataliiaSolotva) [inline cypress/e2e/login.cy.js:31]: Create also separate tests for login with invalid username and for login invalid password

**Review by NataliiaSolotva — CHANGES_REQUESTED** [review]
Check the comments, please.
The next time don't push all changed files. We do not need node_modules directory here)

File: cypress/e2e/login.cy.js (hunk 1 above)
Comment (MarinaT-N) [reply cypress/e2e/login.cy.js]: done

File: cypress/e2e/login.cy.js (hunk 2 above)
Comment (MarinaT-N) [reply cypress/e2e/login.cy.js]: i have created 2 more tests, but i didn't delete the test with all invalid creds.

**Review by VadimMentorQA — APPROVED** [review]
Good job.

**Review by VadimMentorQA — APPROVED** [review]

**Review by VadimMentorQA — APPROVED** [review]

**Pull request closed without merging**